GameData/RealismOverhaul/Engine_Configs/RD107_RD117_Config.cfg

```diff
@@ -744,7 +744,7 @@
 		ignitionReliabilityEnd = 0.983077
 		cycleReliabilityStart = 0.915385
 		cycleReliabilityEnd = 0.983077
-		techTransfer = RD-200:25
+		techTransfer = RD-200:50
```

Comment:
Same question: Why give a full 50% transfer between the RD200 series and the RD107? They're not the same engine at all...